torch/__init__.py

```diff
@@ -1586,6 +1589,37 @@ def __call__(self, model_, inputs_):
         return self.compiler_fn(model_, inputs_, **self.kwargs)
 
 
+################################################################################
+# Exposing torch.export() and related support APIs
+################################################################################
+from ._constraint import Constraint
+
+def export(
+    f: Callable,
+    args: Tuple[Any],
+    kwargs: Optional[Dict[str, Any]] = None,
+    constraints: Optional[List[Constraint]] = None,
+) -> "torch.export_utils.ExportedProgram":
```

Comment:
Hmm what if this is just torch.ExportedProgram?

Reply:
I think either is fine. I don't think users would need to interact with the type of ExportedProgram too much, so not exposing it on torch now.

Reply:
It is odd to expose a public API that returns objects that are not "exposed" - this is part of the contract, just the same

Reply:
For `torch.compile` we saw that people needed to interact with `OptimizedModule` so I agree with @tugsbayasgalan